pages/polling.tsx

```diff
@@ -135,9 +134,11 @@ const PollingOverview = ({ polls, categories }: Props) => {
 
   return (
     <PrimaryLayout shortenFooter={true} sx={{ maxWidth: [null, null, null, 'page', 'dashboard'] }}>
-      <Head>
-        <title>Maker Governance - Polling</title>
-      </Head>
+      <HeadComponent
+        title="Polls"
+        description={`Last poll: ${polls[0].title}. Active Polls: ${activePolls.length}. Total Polls: ${polls.length}. .`}
+      />
+
```

Comment:
I like this a lot. Small suggestions:

`title` should match URL imo: `polling` instead of `polls` 

`Latest` instead of `Last`?